tests/components/setOrderEntry/OrderSetForm.test.jsx

```diff
@@ -28,4 +28,11 @@ describe('Component: OrderSetForm', () => {
     changeSelectedOrderSet();
     expect(renderedComponent.state.displaySets).toEqual('none');
   });
+
+  it('should hide the orderSetForm when cancelOrderSet is called', () => {
```

Comment:
orderSetForm is the whole returned component. This button does not hide it all, it removes a section of it when you set the display to none. Also, you are expecting `display:block`. Please clearly describe the test's purpose and expect display none if you want to hide or expect display block if you want to assert it's presence. Does this make sense?

Reply:
Have you seen [this](https://wiki.openmrs.org/display/docs/Pull+Request+Tips)

Reply:
Thanks donald this has been addressed